components/formats-bsd/src/loci/formats/in/OMETiffReader.java

```diff
@@ -1308,6 +1339,7 @@ private void addSubResolutions() throws IOException, FormatException {
         }
         core.add(s, c);
       }
+      rs.close();
```

Comment:
This line is currently conflicting with changes in https://github.com/openmicroscopy/bioformats/pull/3276